sycl/doc/GettingStartedXilinxFPGA.md

```diff
@@ -17,6 +17,13 @@ https://github.com/Xilinx/XRT/issues/5168 up to its resolution.  In
 the meantime you can always help fixing the bug or install explicitly
 an older Linux kernel package and boot on it.
 
+## What's new?
+
+- 2021/06/24: there is a new HLS target along the OpenCL/SPIR compiler
+  flow for Xilinx FPGA. The HLS target relies on direct LLVM IR
+  feeding and allows finer control by using HLS extensions.
+
+
```

Comment:
All these changes are already in the target branch.

Reply:
Probably a github bug. 
It disappeared after rebasing on sycl/unified/next after the merge of #131